src/sap.ui.core/test/sap/ui/core/qunit/BindingParser.qunit.html

```diff
@@ -659,6 +659,51 @@
 		oModel.setProperty("/blue", true);
 		strictEqual(oIcon.getColor(), "*red*", "one time binding -> value unchanged");
 	});
+	
```

Comment:
Wow, that's a long time ago... But by the look at it: I put tests in here that use an expression binding. With respect to other tests in this module, it seemed to be the right place at this time.